llvm/docs/LangRef.rst

```diff
@@ -15751,7 +15751,9 @@ Semantics:
 """"""""""
 
 This function returns the same values as the libm ``rint`` functions
-would, and handles error conditions in the same way.
+would, and handles error conditions in the same way. The rounding mode
+is assumed to be set to "nearest", so halfway cases are rounded to the
```

Comment:
It's probably worth mentioning that this assumption is because of the default environment (with a link) and mentioning that constrained intrinsics can be used to avoid the assumption.